tests/integrations/mcs/Makefile

```diff
@@ -32,7 +32,7 @@ test: failpoint-enable
 	$(MAKE) failpoint-disable
 
 ci-test-job:
-	CGO_ENABLED=1 go test ./... -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=$(ROOT_PATH)/... github.com/tikv/pd/tests/integrations/mcs
+	CGO_ENABLED=1 go test ./... -timeout=15m -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=$(ROOT_PATH)/... github.com/tikv/pd/tests/integrations/mcs
```

Comment:
why change it?

Reply:
previously mcs/tso needs over 8 mins. After I added tso_proxy_test.go, it needs a little more than 10 minutes. We might need to split mcs/tso into multiple ci chunks later. In this pr, I just simply change it to 15 minutes.